fetch.bs

```diff
@@ -1932,10 +1932,6 @@ message as HTTP/2 does not support them.
 <dfn export for=response id=concept-response-cache-state>cache state</dfn> (the empty string or
 "<code>local</code>"). Unlesss stated otherwise, it is the empty string.
 
-<p class=note>This is intended solely for usage by service workers. [[SW]]
-<!-- If we ever expand the utility of this we need to carefully consider whether filtered responses
-     need to mask it, whether the cache API needs to store it, etc. -->
```

Comment:
I think we should keep this including the comment and instead say something like

> `This is intended for usage by <cite>Service Workers</cite> and <cite>Resource Timing</cite>.`

I guess for Resource Timing we also need opaque responses (that have TAO) to leak this information. I'll have a look if that needs more work.

Reply:
Done